_observing-your-data/alerting/monitors.md

```diff
@@ -18,11 +18,12 @@ redirect_from:
 
 ## Monitor types
 
-The OpenSearch Dashboard Alerting plugin provides four monitor types:
+The OpenSearch Dashboard Alerting plugin provides four basic monitor types, as well as a composite monitor that can utilize any or all monitor types:
 * **per query** – This monitor runs a query and generates alert notifications based on criteria that matches.
 * **per bucket** – This monitor runs a query that evaluates trigger criteria based on aggregated values in the dataset.
 * **per cluster metrics** – This monitor runs API requests on the cluster to monitor its health.
 * **per document** – This monitor runs a query (or multiple queries combined by a tag) that returns individual documents that match the alert notification trigger condition.
+* **composite monitor** — The composite monitor allows you to chain the execution of multiple monitors of different types into a single flow, where the analysis and findings of one monitor becomes the input or source data of the next monitor in the chain. See [Composite monitors]({{site.url}}{{site.baseurl}}/notifications-plugin/index) for information on creating and using these types of monitors.
```

Comment:
We don't chain the output of one monitor for the other. For now, we execute the selected monitors and then based on the trigger condition we decide whether the added action should be executed.
@eirsep Can you help here?